src/control/SConscript

```diff
@@ -68,13 +68,13 @@ def install_go_bin(denv, gosrc, gopath, libs, name, install_name):
         src.extend(libs)
     denv.Command(bin_path, src,
                  cmd_build(path, gosrc,
-                           "go install %s" % install_src))
+                           'go install -ldflags "-X main.daosVersion=%s" %s' % (DAOS_VERSION, install_src)))
     denv.InstallAs(install_bin, bin_path)
 #pylint: enable=too-many-arguments
 
 def scons():
     """Execute build"""
-    Import('env', 'prereqs')
+    Import('env', 'prereqs', 'DAOS_VERSION')
```

Comment:
Is this required ? I did not have to add this in src/utils/SConscript for daos tool, as it is defined and passed from main SConstruct

Reply:
If I don't add that, then I get this error when building:
```
NameError: global name 'DAOS_VERSION' is not defined:
  File "/home/mjmac/go/src/github.com/daos-stack/daos/SConstruct", line 310:
    scons()
...
etc.
```

Reply:
Ah ok, may be it is because you refer to it in line #71 ...

Reply:
Ah ok, could be due to the fact you reference it in line 71 ...